acceptance/tests/options/ignore_cache.rb

```diff
@@ -0,0 +1,53 @@
+test_case "--ignore-cache command-line option causes the fact cache to be ignored" do
+  require 'facter/acceptance/user_fact_utils'
+  extend Facter::Acceptance::UserFactUtils
+
+  # the kernel fact should be resolvable on ALL systems
+  cached_fact_name = "kernel"
+
+  config = <<-FILE
+  cli : { debug : true }
+  facts : { ttls : [ "kernel" : 30 minutes ] }
```

Comment:
Can we also test expiration by modifying mtime on the cache file (make it appear older than 30 minutes) and trying again?